semantic/src/site/globals/site.overrides

```diff
@@ -319,6 +329,51 @@ a:hover {
   max-width: 150px;
 }
 
+/*---------------------------------------
+      Review page
+----------------------------------------*/
+
+// NOTE: The review page shares a lot of styles with Application Summary
+
+.review-comment-area {
+  background-color: @reviewCommentBackground;
+  margin-top: 10px;
+  margin-left: -20px;
```

Comment:
I don't understand why the negative margins here I think I was seeing something weird because of this when trying to update the comment to also show Consolidator comments in between the Reviewer comment and Application response

Reply:
The Element wrapper has some padding, which means that when `review-comment-area` sits inside it, there's white space outside it. The negative margin overcomes this while still allowing the parent component to have sensible padding, otherwise it looks like this:
![Screen Shot 2021-05-10 at 12 25 46 PM](https://user-images.githubusercontent.com/5456533/117592027-035e1500-b18b-11eb-9a02-70d6b846ee98.png)

Reply:
Ahhh, alright : )